cookbooks/bcpc-hadoop/attributes/default.rb

```diff
@@ -127,3 +127,9 @@
 default[:bcpc][:hadoop][:hdfs][:ldap][:password] =  nil
 default[:bcpc][:hadoop][:hdfs][:ldap][:search][:filter][:user]="(&(objectclass=user)(sAMAccountName={0}))"
 default[:bcpc][:hadoop][:hdfs][:ldap][:search][:filter][:group]="(objectClass=group)"
+#
+# Some log4j defaults
+#
+
+default[:bcpc][:hadoop][:log4j][:max_file_size] = "100MB"
+default[:bcpc][:hadoop][:log4j][:max_backups] = 10
```

Comment:
These should be used in:
* [hb_log4.properties.erb](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/hb_log4j.properties.erb)
* [hdp_log4j.properties](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/hdp_log4j.properties.erb)
* [httpfs-log4j.properties.erb](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/httpfs-log4j.properties.erb)
* [hue_log4j.properties.erb](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/hue_log4j.properties.erb)
* [hv_hive-exec-log4j.properties](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/hv_hive-exec-log4j.properties.erb)
* [hv_hive-log4j.properties.erb](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/hv_hive-log4j.properties.erb)
* [ooz_oozie-log4j.properties.erb](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/ooz_oozie-log4j.properties.erb)
* [zk_log4j.properties.erb](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/templates/default/zk_log4j.properties.erb)